analyzer/decrypt_crypto.cc

```diff
@@ -330,6 +330,13 @@ hilti::rt::Bytes decrypt_crypto_payload(
     const hilti::rt::Bool &from_client)
 {
 
+    if ( payload_length < 20 )
+        throw hilti::rt::RuntimeError(hilti::rt::fmt("payload too small %ld < 20", (size_t)payload_length));
```

Comment:
The cast (in particular a C-style one) is not necessary here I think.

```suggestion
        throw hilti::rt::RuntimeError(hilti::rt::fmt("payload too small %ld < 20", payload_length));
```

If you need it please use a `static_cast` instead.